core/src/main/java/org/dcsa/conformance/core/toolkit/JsonToolkit.java

```diff
@@ -42,7 +42,7 @@ public static JsonNode templateFileToJsonNode(
     }
     if (replacements != null)
       replacements.forEach(
-          (key, value) -> jsonString.set(jsonString.get().replaceAll(key, value)));
+          (key, value) -> jsonString.set(jsonString.get().replace(key, value)));
```

Comment:
This change can be discussed, but I did not see that we're using regex to replace data, if so, I'll revert it.

Reply:
Yes, I can remove it, when we do like to support regex in the key searching. Because the current implementation with `replaceAll` supports regex and `replace` does not.

Reply:
Ok, I'll revert this change, so we can use regex for search & replace key/values inside sample data.

Reply:
Then I'll make it `replace`, so regex is not taken into account, if I understood you correctly. I also thought it might cause side effects when user input also contains regex. 

Yeah, your confusing happens quite often by many people, because if feels like `replace` is only doing 1 replacement and `replaceAll` is taking multiple hits into consideration.. But Java is not going to change that, because of backwards compatibility issues..